src/modules/Dashboard/features/Respondents/Popups/EditRespondentPopup/EditRespondentPopup.tsx

```diff
@@ -86,7 +87,7 @@ export const EditRespondentPopup = ({
       onClose={onCloseHandler}
       onSubmit={handleSubmit(submitForm)}
       disabledSubmit={isLoading}
-      title={t('editRespondent')}
+      title={t('editParticipant')}
```

Comment:
I noticed that the JIRA ticket says there isn't a design in Figma for the Edit Participant screen, but I didn't have trouble [finding it](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1714/files) myself:

<img width="560" alt="Edit Participant" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/df261900-dbfa-4c7b-91ad-819eeaec9771">

It seems there's a couple discrepancies here:
- it should be bit narrower than the default Modal (`width="56"` same as in `UpgradeAccountPopup`)
- the Cancel button should be specified using the `leftBtn*` props instead of `secondBtn*`

Also might not hurt to just edit the JIRA ticket to include the correct screenshot and remove that last bullet in the "Notes for testing" section.